tests/Tests/400__ods_dashboard/400__ods_dashboard.robot

```diff
@@ -124,3 +130,13 @@ Verify JupyterHub Card CSS Style
     CSS Property Value Should Be    locator=${SIDEBAR_TEXT_CONTAINER_XP}/h1
     ...    property=font-family    exp_value=RedHatDisplay
     ...    operation=contains
+
+Test Setup For Login Page
+    Set Library Search Order  SeleniumLibrary
+    Open Browser  ${ODH_DASHBOARD_URL}  browser=${BROWSER.NAME}  options=${BROWSER.OPTIONS}
+
+Check OpenShift Login Visible
+    ${result}=  Is OpenShift Login Visible
+    IF  ${result}=='false'
+        FAIL    OpenShift Login Visible
```

Comment:
I think message should be **"OpenShift Login Is Not Visible"**

Reply:
Yes. I'll update it.